### PR TITLE
unfreeze: move to 1.0

### PR DESCRIPTION
**Pull request by pared**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Related to #1255

### CODE REVIEW

File: content/docs/command-reference/freeze.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ First, let's create a simple DVC-file:
 ```dvc
 $ echo foo > foo
 $ dvc add foo
-$ dvc run -d foo -o bar -n make_copy cp foo bar
+$ dvc run -n make_copy -d foo -o bar cp foo bar
```
Comment (pared) [inline content/docs/command-reference/freeze.md:47]: https://github.com/iterative/dvc.org/pull/1488/files#r444373602

File: content/docs/command-reference/unfreeze.md [hunk 2]
```diff
@@ -40,51 +40,38 @@ First, let's create a simple DVC-file:
...
-$ dvc unfreeze bar.dvc
-$ dvc status
-
-  bar.dvc
-          deps
-                  changed:  foo
-  foo.dvc
-          outs
-                  changed:  foo
+$ dvc unfreeze make_copy
```
Comment (pared) [inline content/docs/command-reference/unfreeze.md:76]: https://github.com/iterative/dvc.org/pull/1488/files#r444374258

I think there is no need to put the statement in the end, as it is stated in the beginning:
"Run `dvc unfreeze` to unfreeze it back."

File: content/docs/command-reference/freeze.md [hunk 3]
```diff
@@ -77,12 +77,4 @@ Run `dvc unfreeze` to unfreeze it back:
 
 ```dvc
 $ dvc unfreeze make_copy
```
Comment (pared) [inline content/docs/command-reference/freeze.md:79]: https://github.com/iterative/dvc.org/pull/1488/files#r444374258

I think there is no need to put the statement in the end, as it is stated in the beginning:
"Run dvc unfreeze to unfreeze it back."

**Comment by jorgeorpinel** [comment]
Oops, looks like a duplicate of #1493 ?

**Comment by pared** [comment]
My bad forgot te check existing PR's today, sorry @jorgeorpinel. Will find some other article to mess around with!

**Comment by jorgeorpinel** [comment]
No sorry, it looks like I took over your planned docs contributions! If you could help us review that other one that would be great too :) maybe some of your comments up here apply? Thanks!

**Pull request closed without merging**